collection/templates/_collection.coffee

```diff
@@ -1,3 +1 @@
-root = exports ? this
-
-root.<%= name.charAt(0).toUpperCase() + name.slice(1).toLowerCase() %> = new Meteor.Collection('<%= name.toLowerCase() %>')
\ No newline at end of file
+@<%= name.charAt(0).toUpperCase() + name.slice(1).toLowerCase() %> = new Meteor.Collection('<%= name.toLowerCase() %>')
```

Comment:
I found it more convenient to make the collection global like this, if you do not like it i can revert this change.